compiler/src/dotty/tools/dotc/transform/SymUtils.scala

```diff
@@ -107,6 +108,13 @@ object SymUtils:
 
     def isGenericProduct(using Context): Boolean = whyNotGenericProduct.isEmpty
 
+    /** Is a case class for which mirrors support access to default arguments.
+     */
+    def mirrorSupportsDefaultArguments(using Context): Boolean = self.isClass && {
+      val d = self.asClass.classDenot
+      TastyFormat.isVersionCompatible(28, 4, 1, d.tastyMajorVersion, d.tastyMinorVersion, d.tastyExperimentalVersion)
```

Comment:
the thing that's complex is that eventually this will need to be a comparison to `28, 4, 0`, aka a stable version, but then we can't test that until TastyFormat changes

Reply:
potentially you could relax the comparison in this case to only check the major/minor